homeassistant/config.py

```diff
@@ -146,6 +147,9 @@ class ConfigExceptionInfo:
     integration_link: str | None
 
 
+_T = TypeVar("_T")
```

Comment:
This seems unused.

Reply:
Fixed by https://github.com/home-assistant/core/pull/108495